backend/src/main/java/com/soen341/instagram/exception/account/BadRequestException.java

```diff
@@ -0,0 +1,13 @@
+package com.soen341.instagram.exception.account;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+public class BadRequestException extends RuntimeException
+{
+	public BadRequestException()
+	{
```

Comment:
As mentioned in another comment, being able to give more precise messages for a bad request might be a good way to debug